CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ option(CARE_ENABLE_MANAGED_PTR "Enable managed_ptr aliases, tests, and reproduce
 option(CARE_DISABLE_RAJAPLUGIN "Disable use of the RAJA plugin. WILL ALSO DISABLE MEMORY MOTION." OFF)
 option(CARE_ENABLE_EXTERN_INSTANTIATE "Enable extern instantiation of template functions" OFF)
 option(CARE_ENABLE_GPU_SIMULATION_MODE "Enable GPU simulation mode." OFF )
+option(CARE_ENABLE_LOOP_FUSER "Enable the loop fuser" OFF)
```

Comment:
I think we should use cmake_dependent_option.

Add this at around line 11: `include(CMakeDependentOption)`

Then you can use something like this here: `cmake_dependent_option(CARE_ENABLE_LOOP_FUSER) "Enable the loop fuser" ON "ENABLE_CUDA" OFF)`

Reply:
Done.